api/handler/v1beta1/job_spec.go

```diff
@@ -9,6 +9,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/gofrs/uuid"
```

Comment:
This should be importing google uuid module
`github.com/google/uuid` instead of this.